osu.Game.Rulesets.Osu/Mods/OsuModHidden.cs

```diff
@@ -83,5 +88,18 @@ protected void ApplyHiddenState(DrawableHitObject drawable, ArmedState state)
                     break;
             }
         }
+
+        public void ReadFromConfig(OsuConfigManager config)
+        {
+            increaseFirstObjectVisibility = config.GetBindable<bool>(OsuSetting.IncreaseFirstObjectVisibility);
+            //This starts the process of applying the mod effects. We start it here since this is the last void called.
+            applyMod();

```

Comment:
this shouldn't be called here. this call itself should be before other application calls.